specs/AdminPermissionsNewUsers.spec

```diff
@@ -0,0 +1,34 @@
+AdminPermissionsNewUsers
+===================================
+
+AdminPermissionsNewUsers
+------------------------
+
+tags: admin_permissions
+
+Setup of contexts
+* Basic Configuration - setup
+* With "1" live agents - setup
+* Capture go state "AdminPermissionsNewUsers" - setup
+
+* Enable security with password file
+* Login as "admin" - setup
+
+Verify if user is admin
+* VerifyIfUserIsAdmin 
```

Comment:
@rahulpargaonkar we can rename this steps to VerifyIfUserHasRole as the implementation can be reused later